bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -191,6 +197,30 @@ for veto_file, veto_segment_name in zip(args.veto_file, args.veto_segment_name):
         tparam = tparam[retain]
     logging.info('%i trigs left after vetoing with %s' %
                                                        (len(stat), veto_file))
+#now include gating veto
+if args.gating_veto_windows:
+    gating_veto = args.gating_veto_windows[args.ifo].split(',')
+    gveto_before = float(gating_veto[0])
+    gveto_after = float(gating_veto[1])
+    if gveto_before > 0 or gveto_after < 0:
+        raise ValueError("Gating veto window values must be negative before "
+                         "gates and positive after gates.")
+    if gveto_before == 0 and gveto_after == 0:
+        pass
+    else:
+        gate_times = np.unique(trigf[args.ifo + '/gating/auto/time'][:])
+        gveto_segs = events.veto.start_end_to_segments(gate_times + gveto_before,
+                                                       gate_times + gveto_after).coalesce()
+        all_segments -= gveto_segs
+        gveto_retain_idx = events.veto.indices_outside_times(time, gate_times + gveto_before,
```

Comment:
could this be split into two lines on the comma between `time` and `gate_times`?